lib/winrm-elevated/scripts/elevated_shell.ps1

```diff
@@ -94,6 +94,9 @@ do {
   $err_cur_line = SlurpOutput $err_file $err_cur_line 'err'
 } while (!($registered_task.state -eq 3))
 
+del $out_file
+del $err_file
```

Comment:
Thinking out loud here... I've seen a case where the WinRM service gets bounced and a command is retried and the log file is "locked by another process". I don't think that should be able to happen now that we're creating unique log files. I was going to suggest `Remove-Item $out_file -ErrorAction SilentlyContinue`, but that seems pointless now.